src/pages/organizations.js

```diff
@@ -70,37 +72,47 @@ const SectorTemplate = ({ data }) => {
 }
 
 export const query = graphql`
-  query SectorPageQuery($slug: String) {
-    airtable(table: { eq: "Sectors" }, data: { Slug: { eq: $slug } }) {
-      data {
-        Name
-        Organizations {
-          data {
-            Name
-            Homepage
-            About
-            Slug
-            Tags,
-            Tagline,
-            City,
-            State_Province,
-            Country,
-            Organization_Type,
-            Headcount,
-            Logo {
-              localFiles {
-                childImageSharp {
-                  fixed(width: 64, height: 64, fit: CONTAIN, background: "white") {
-                    ...GatsbyImageSharpFixed
-                  }
+  query OrganizationsPageQuery($slugRegex: String) {
+    organizations: allAirtable(filter: {table: {eq: "Organizations"}, data: { Sector: { elemMatch: { data: { Slug: { regex: $slugRegex} } } } } }) {
```

Comment:
👍 querying the organizations table makes sense.

If we need to limit the number of rows shown on initial page load we can always add a limit.